README.md

```diff
@@ -14,8 +14,33 @@ has only read-access to the database, a writer has read- and write-access.
 A database can be accessed either by any number of readers or by exactly one
 writer at the same time.
 
+## Installing the required libraries.
+
+### On Debian/Ubuntu
+
+```
+sudo apt-get update
+sudo apt-get install libgdbm-dev
+```
+
+### On Redhat/Fedora  Yum or DNF
+```
+sudo yum install gdbm-devel
+
+sudo dnf install gdbm-devel
+```
```

Comment:
Aligning the "On Debian/Ubuntu" (upstream/downstream distro) above, I would suggest "On Fedora/Red Hat Enterprise Linux" or "On Fedora/RHEL" (upstream/downstream distro).

The OS name is [RHEL](https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux). It's different from old unused OS, [Red Hat Linux](https://en.wikipedia.org/wiki/Red_Hat_Linux).

Nowadays the `yum` is just alias for the `dnf`. The `dnf` is available in RHEL 8 and later versions.

```
$ cat /etc/fedora-release
Fedora release 38 (Thirty Eight)

$ ls -l /bin/yum
lrwxrwxrwx. 1 root root 5 Jul 27 02:00 /bin/yum -> dnf-3*
```

So, I would suggest the `dnf` as first, and `yum` as second.

```
sudo dnf install gdbm-devel

sudo yum install gdbm-devel
```